src/errors/specification-file.ts

```diff
@@ -22,3 +23,31 @@ export class SpecificationURLNotFound extends SpecificationFileError {
     this.message = `Unable to fetch specification file from url: ${URL}`;
   }
 }
+
+type From = 'file' | 'url' | 'context'
+
+export class ErrorLoadingSpec extends Error {
+  private readonly errorMessages = {
+    default: NO_CONTEXTS_SAVED
+  }
+  constructor(from?: From, param?: string) {
+    super();
+    if (from === 'file') {
+      this.name = 'error loading specification from file';
```

Comment:
instead of `loading specificaton` always write `loading AsyncAPI document` in all error messages